packages/gallery/src/components/gallery/proGallery/galleryContainerExtraNew.js

```diff
@@ -390,20 +390,18 @@ export class GalleryContainer extends React.Component {
     });
   }
 
-  createCssLayoutsIfNeeded(layoutParams, isApproximateWidth = false) {
+  createCssLayoutsIfNeeded(layoutParams) {
```

Comment:
Wasn't this commented out in the blueprints flow?

Reply:
W/o this there is no this.layoutCss and nothing renders. If this should be removed how will it work?